backend/tests/integration/submissions/test_submissions_views.py

```diff
@@ -2277,3 +2278,63 @@ def test_new_institution_submission_with_empty_data(
 
     assert response.status_code == 400
     assert response.json["message"][0] == "Institution is missing a value or values."
+
+
+REQUIRED_JOURNAL_RECORD_DATA = {
+    "$schema": "http://localhost:5000/schemas/records/journals.json",
+    "short_title": "RoMP",
+    "journal_title": {'title': 'Reviews of Modern Physics'},
+    "_collections": ["Journals"]
+}
+JOURNAL_FORM_DATA = {
+    "short_title": "RoMP",
+    "journal_title": {'title': 'Reviews of Modern Physics'},
+}
+
+
+@pytest.mark.parametrize(
+    "form_data,expected_record_data",
+    [
+        (deepcopy(JOURNAL_FORM_DATA), REQUIRED_JOURNAL_RECORD_DATA),
+    ],
+)
```

Comment:
no need to parametrize, in the test you could just create a record:

```python
record =  create_record_factory("jou")
```

and have the form_data inside the test